Dockerfile

```diff
@@ -19,17 +19,17 @@ RUN make build
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.10
 
 
-COPY --from=builder /workspace/bin/ciam-rebac /usr/local/bin/
+COPY --from=builder /workspace/bin/relations-api /usr/local/bin/
 COPY --from=builder /workspace/configs/config.yaml /usr/local/bin/
 
 EXPOSE 8000
 EXPOSE 9000
 
 USER 1001
 
-ENTRYPOINT ["/usr/local/bin/ciam-rebac","-conf","/usr/local/bin/config.yaml"]
+ENTRYPOINT ["/usr/local/bin/relations-api","-conf","/usr/local/bin/config.yaml"]
```

Comment:
Should our binary have "kessel" in it? (or "ksl" if you want to keep it short)

e.g. kessel-relations or ksl-relations

Reply:
name of binary: kessel-relations or ksl-relations

Decision: kessel-relations (for the server binary)